Jenkinsfile

```diff
@@ -1 +1,5 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
+buildPlugin(configurations: [
+    [ platform: "linux", jdk: "8"],
+    [ platform: "windows", jdk: "8"],
+    [ platform: "linux", jdk: "11", jenkins: "2.176.4", javaLevel: "8" ]
```

Comment:
Any reason you aren’t testing a more recent lts here?

Reply:
Trying to keep things simple-ish in this PR.